service/server.py

```diff
@@ -151,13 +151,27 @@ def sign_out():
 def display_title(title_ref):
 
     title = get_register_title(title_ref)
+    page_number = int(request.args.get('page_number', 1))
+    search_term = request.args.get('search_term', title_ref)
     if title:
         # If the title was found, display the page
         LOGGER.info(
             "VIEW REGISTER: Title number {0} was viewed by '{1}'".format(title_ref,
                                                                          current_user.get_id()))
-        return render_template('display_title.html', title=title,
-                               username=current_user.get_id())
+        return render_template('display_title.html',
+                               title=title,
+                               username=current_user.get_id(),
+                               search_term=search_term,
+                               page_number=page_number,
+                               breadcrumbs=[
+                                   {"text": "Find a title", "url": url_for('find_titles')},
+                                   {"text": "Search results",
```

Comment:
I wouldn't expect a 'Search results' breadcrumb if you are displaying the title and not coming from a search?